docs/ref/applications.txt

```diff
@@ -148,8 +130,12 @@ configuration::
         # ...
     ]
 
-Again, defining project-specific configuration classes in a submodule called
-``apps`` is a convention, not a requirement.
+This example shows project-specific configuration classes located in a
+submodule called ``apps.py``. This is a convention, not a requirement.
+:class:`~django.apps.AppConfig` subclasses may be defined anywhere.
+
+In this situation, :setting:`INSTALLED_APPS` must contain the dotted path to
```

Comment:
"In this situation" — what situation? The one in the last paragraph? (But which... the actual example, or the subjunctive `may be defined anywhere`?) 

> Where the AppConfig is defined somewhere other than in `apps.py`...